app/views/handlers/individual_response.py

```diff
@@ -0,0 +1,335 @@
+            "individual_response.request_individual_response",
+            list_item_id=self._list_item_id,
+        )
+
+        return render_template(
+            "individual_response/how",
+            language=self._language,
+            content=self.get_context(),
+            previous_location_url=previous_location_url,
+        )
+
+
+class IndividualResponsePostAddressConfirmHandler(IndividualResponseHandler):
+    block_definition = {
+        "type": "IndividualResponse",
+        "question": {
+            "type": "Question",
+            "id": "individual-response-post-confirm",
+            "title": {
+                "text": lazy_gettext(
+                    "Do you want to send an individual access code to <em>{person_name}</em> by post?"
+                ),
+                "placeholders": [
+                    {
+                        "placeholder": "person_name",
+                        "transforms": [
+                            {
+                                "arguments": {
+                                    "delimiter": " ",
+                                    "list_to_concatenate": {
+                                        "identifier": ["first-name", "last-name"],
+                                        "source": "answers",
+                                    },
+                                },
+                                "transform": "concatenate_list",
+                            }
+                        ],
+                    }
+                ],
+            },
+            "description": {
+                "text": lazy_gettext(
+                    "<p>A letter with an indivdual access code will be sent to:</p>"
+                    "<strong>Individual Resident</strong><br>"
+                    "{display_address}"
+                ),
+                "placeholders": [
+                    {
+                        "placeholder": "display_address",
+                        "value": {
+                            "identifier": "display_address",
+                            "source": "metadata",
+                        },
+                    }
+                ],
+            },
+            "guidance": {
+                "contents": [
+                    {
+                        "description": lazy_gettext(
+                            "For security reasons we can only send access codes to an "
+                            "unnamed resident at the registered household address"
+                        )
+                    }
+                ]
+            },
+            "answers": [
+                {
+                    "type": "Radio",
+                    "id": "individual-response-post-confirm-answer",
+                    "mandatory": True,
+                    "options": [
+                        {
+                            "label": lazy_gettext("Yes, send the access by post"),
+                            "value": "Yes, send the access by post",
```

Comment:
The label and value could be a variable. Given we don't validate these in-code schemas, it would help protect against someone changing the label but not the value and vice versa.

Reply:
For every label and value?